packages/service-clients/odsp-client/api-report/odsp-client.alpha.api.md

```diff
@@ -7,6 +7,12 @@
 // @beta
 export type IOdspAudience = IServiceAudience<OdspMember>;
 
+// @beta @sealed
+export interface IOdspContainerServicesEvents extends IEvent {
+    (event: "readOnlyStateChanged", listener: (readonly: boolean) => void): void;
+    (event: "sensitivityLabelChanged", listener: (sensitivityLabelsInfo: string) => void): void;
```

Comment:
Nit: Probably try to keep naming parallel between the method and event (either `sensitivityLabelsInfoChanged` or `getSensitivityLabel()`)